spring-cloud-stream/src/main/java/org/springframework/cloud/stream/config/ChannelBindingAdapterRunner.java

```diff
@@ -17,16 +17,15 @@
 package org.springframework.cloud.stream.config;
 
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.boot.CommandLineRunner;
+import org.springframework.boot.ApplicationArguments;
+import org.springframework.boot.ApplicationRunner;
 import org.springframework.cloud.stream.adapter.ChannelBindingAdapter;
-import org.springframework.context.annotation.Configuration;
 
 /**
  * @author Dave Syer
  *
  */
-@Configuration
-public class LifecycleConfiguration implements CommandLineRunner {
+public class ChannelBindingAdapterRunner implements ApplicationRunner {
```

Comment:
Yeah, both works fine in this context. It is just that name `CommandLine` made me think otherwise. I can revert the changes to use `CommandLineRunner`. Shall I?